### PR TITLE
added 256 color support

### PR DESCRIPTION
**Pull request by Qix-**
In response to #1.

WDYT @sindresorhus @jbnicolai

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -2,14 +2,19 @@
...
 		'  $ supports-color',
+		'  $ supports-color --256',
+		'  $ supports-color --16m',
 		'',
 		'Exits with code 0 if color is supported and 1 if not'
 	]
 });
 
-process.exit(supportsColor ? 0 : 1);
+var level = hasFlag('256') ? 'has256' : hasFlag('16m') ? 'has16m' : 'hasBasic';
```
Comment (sindresorhus) [inline cli.js]: No need for `hasFlag` as `meow` parses the CLI arguments: https://github.com/sindresorhus/meow#usage

**Comment by Qix-** [comment]
@sindresorhus done.

File: cli.js [hunk 2]
```diff
@@ -3,13 +3,18 @@
...
 var supportsColor = require('supports-color');
 
-meow({
+var cli = meow({
 	help: [
 		'Example',
 		'  $ supports-color',
+		'  $ supports-color --256',
+		'  $ supports-color --16m',
 		'',
```
Comment (sindresorhus) [inline cli.js:19]: We should probably explain the flags in an `Options` section. Like: https://github.com/sindresorhus/fkill-cli#usage

The readme also needs to be synced with the new `--help` output.

**Comment by Qix-** [comment]
@sindresorhus sorry this took me so long; work has sucked the last week. That should work.

**Comment by sindresorhus** [comment]
No worries, I've been mostly afk the past week anyways.

File: cli.js [hunk 3]
```diff
@@ -3,13 +3,25 @@
...
 
-meow({
+var cli = meow({
 	help: [
+		'Usage',
+		'  $ supports-color [--<256|16m>]',
+		'',
+		'Options',
+		'  --256     Check for 256 color support',
+		'  --16m     Check for 16 million (truecolor) support',
```
Comment (sindresorhus) [inline cli.js:13]: Just curious, do you know if there's a standard for how many spaces the flag and description should have? I'm never sure.

Comment (Qix-) [inline cli.js:13]: There isn't to my knowledge. I always figured to keep it inside of 80 characters, but I see that broken all the time.

**Comment by sindresorhus** [comment]
Super! :)

File: cli.js (hunk 3 above)
Comment (sindresorhus) [inline cli.js:13]: Why keep it to 80 though? Most terminals supports longer and soft wraps.

Comment (Qix-) [inline cli.js:13]: Soft wraps cause words to break. Not ideal for documentation, I suppose. I'm pretty indifferent on the widths of documentation.

**Pull request merged**